js-api-spec/value/color/color-4-channels.test.ts

```diff
@@ -0,0 +1,295 @@
+// Copyright 2023 Google Inc. Use of this source code is governed by an
+// MIT-style license that can be found in the LICENSE file or at
+// https://opensource.org/licenses/MIT.
+
+// General space-specific info and construction
+
+import {SassColor} from 'sass';
+import type {ChannelNameXyz, ColorSpaceXyz, KnownColorSpace} from 'sass';
+import {List} from 'immutable';
+
+import {spaces} from './spaces';
+import {channelCases, channelNames} from './utils';
+import {skipForImpl} from '../../utils';
+
+const spaceNames = Object.keys(spaces) as KnownColorSpace[];
+
+describe('Color 4 SassColor Channels', () => {
+  spaceNames.forEach(spaceId => {
+    const space = spaces[spaceId];
+
+    describe(space.name, () => {
+      let color: SassColor;
+      beforeEach(() => {
+        color = space.constructor(...space.pink);
+      });
+
+      describe('channelsOrNull', () => {
+        it('returns a list', () => {
+          expect(color.channelsOrNull).toFuzzyEqualList(space.pink);
+          expect(color.channelsOrNull.size).toBe(space.channels.length);
+          expect(List.isList(color.channelsOrNull)).toBeTrue();
+        });
+        it('returns channel value or null, excluding alpha', () => {
+          const pinkCases = channelCases(...space.pink);
+          pinkCases.forEach(channels => {
+            const _color = space.constructor(...channels);
+            expect(_color.channelsOrNull).toFuzzyEqualList(
+              channels.slice(0, 3)
+            );
+          });
+        });
+      });
+      describe('channels', () => {
```

Comment:
Style nit: please leave an empty line between test cases and groups.

Reply:
Addressed in https://github.com/sass/sass-spec/pull/1949/commits/64d225a9e2ff5d5c669cfba09fd341d898bb82f7.